apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -143,12 +143,8 @@ async function syncAccount(
   );
 
   // Match pending transfers
-  const nowS = Date.now() / 1e3;
-  const maxPendingAgeS = 5 * 60; // 5 mins
   const stillPending = oldPending.filter(
-    (t) =>
-      syncFindSameOp(t, recentTransfers) == null &&
-      nowS - t.timestamp < maxPendingAgeS
```

Comment:
deleted this conditional, seems like it should never happen (or if it does, its a bug so we should not hide it?)

Reply:
this handles the case where bundler fails to include an op (or it gets reorg-d out or reverts for some reason)

avoids having a op that stays pending indefinitely

find to remove for now -  we can reintroduce this in a more sound way by
- using the `validUntil` on all outgoing ops
- saving that expiration date on the pending op in account storage
- in sync, remove pending ops past their expiration (but only after we've successfully loaded sync data from API--they *can* stay pending indefinitely while you're offline, so they resolve once your back online)

Reply:
moved to its own issue #325 but will bring this back in this PR so it’s not weird in the short term